data/remnant missions.txt

```diff
@@ -1331,6 +1331,26 @@ mission "Remnant Return the Samples Compatibility Patch"
 		fail
 
 
+# "This mission is intended to serve as a notification that one timer has elapsed and how long to wait until the missions start again."
+mission "Remnant: Return the Samples 3 Exploration Hint"
+	name "Return the Samples Timer Elapsed"
+	description "According to your commlink, Plume has reached a milestone in his research."
+	landing
+	source Aventine
+	to offer
+		has "event: remnant: return the samples timer"
+	on offer
+		event "remnant: wait for plume" 7 14
+		conversation
+			`As you disembark from <ship> you almost run into Plume as he strides across the landing area. "Captain <first>, I didn't see you there!" You glance back at the bulk of your ship that you just stepped out of and look at him curiously. His chant takes on a slightly apologetic tone "I've been lost in thought thinking about everything we have discovered lately, especially the most recent recordings you made. I still have a lot to analyze, though." He pauses, looking thoughtful. "That being said, I will probably have more work for you in a week or two."`
```

Comment:
Why am I glancing back at my ship? Did he run past me and to it for some reason?
Missing period after "apologetic tone".
```suggestion
			`As you disembark from <ship> you almost run into Plume as he strides across the landing area. "Captain <first>, I didn't see you there!" You glance back at the bulk of your ship that you just stepped out of and look at him curiously. His chant takes on a slightly apologetic tone. "I've been lost in thought thinking about everything we have discovered lately, especially the most recent recordings you made. I still have a lot to analyze, though." He pauses, looking thoughtful. "That being said, I will probably have more work for you in a week or two."`
```

Reply:
I meant that comment about looking back at the ship along the lines of "You just landed and got off your ship, and plume almost runs into you; so you are looking back at your ship thinking "What, is my ship cloaked? Shouldn't it be obvious that I'm here?" 

But yeah, it doesn't come across really well.